cve_bin_tool/checkers/apache.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for apache
```

Comment:
```suggestion
CVE checker for Apache HTTP Server
```
You'd also have to change the filename to apache_http_server.py